core/src/main/resources/lib/layout/copyButton.jelly

```diff
@@ -26,9 +26,9 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:i="jelly:fmt" xmlns:x="jelly:xml">
   <st:documentation>
-    Button that copies text into the user's clipboard upon click
+    Button that copies text into the user's clipboard upon click, either <code>text</code> or <code>ref</code> is required.
 
-    <st:attribute name="text" use="required">
```

Comment:
If that is required, the `use="required"` argument must be retained and added to the ref below.